InternTrack.Portal.Web.Tests.Unit/Services/DeleteMe.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace InternTrack.Portal.Web.Tests.Unit.Services
```

Comment:
This should be under Foundations

Reply:
@glhays Do you mean the Service folder or the DeleteMe.cs either way that's how he had it set up in the video

Reply:
@Pwongchaiya Ok then... just mark as resolved, will be deleted soon when a proper test is initiated.